vsprojects/vcxproj/grpc_csharp_ext/grpc_csharp_ext.vcxproj

```diff
@@ -101,6 +101,7 @@
       <SubSystem>Windows</SubSystem>
       <GenerateDebugInformation Condition="!$(Jenkins)">true</GenerateDebugInformation>
       <GenerateDebugInformation Condition="$(Jenkins)">false</GenerateDebugInformation>
+      <OutputFile>$(OutDir)$(TargetName)-x64$(TargetExt)</OutputFile>
```

Comment:
please note that this file is autogenerated, you'd need to change to template under templates/ to make this work properly.

Also, a change to the .nuspec file would be necessary to make sure all the libraries are properly copied to the output folder once someone imports gRPC as a nuget packages and builds his project.